### ISSUE TITLE
bug at models.js on POS

### ISSUE THREAD

**Issue by yelizariev** [odoo/odoo#8983]
load_model function uses context as kwargs if ids parameter is provided:

```
            if( model.model ){
                if (model.ids) {
                    records = new Model(model.model).call('read',[ids,fields],context);
```

https://github.com/odoo/odoo/blob/358e285fd8ed75ac322213bd1d3382ff7fc28eba/addons/point_of_sale/static/src/js/models.js#L502

While it doesn't make errors as point_of_sale doesn't use **ids** along with **context** in PosModel.models, it could cause a misunderstanding in custom POS modules 

Possible fix:

```
            if( model.model ){
                if (model.ids) {
                    records = new Model(model.model).call('read',[ids,fields],{'context':context});
```

related commit: https://github.com/odoo/odoo/commit/1d23c24d79a0759d2832ffc2f10b3469470ce0d7 by @ged-odoo

**Comment by rafaelbn** [comment]
cc @pedrobaeza @antespi @Yajo

**Comment by rafaelbn** [comment]
@yelizariev it's much more effective if you make a PR to Odoo fixing it.
Also you can make a PR to https://github.com/OCA/OCB fixing the problem and have it solve it

**Comment by rafaelbn** [comment]
:+1:

**Comment by sle-odoo** [comment]
Hi,
I'm closing this issue as i assume it is not really a bug according to your first comment.

Regards

**Comment by yelizariev** [comment]
@sle-odoo ok
Also, it was fixed in Odoo 9.0 during refactoring https://github.com/odoo/odoo/commit/1d23c24d79a0759d2832ffc2f10b3469470ce0d7
